src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,50 @@
 import React from 'react';
+import { Movie } from '../Movie';
 
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
-);
+type Props = {
+  movieFromServer: Omit<Movie, 'imdbId'>,
```

Comment:
why omit?
do not call it `movieFromServer`, avoid `fromServer`
```suggestion
  movie: Movie;
```